src/views/Tournament/Tournament.tsx

```diff
@@ -1191,13 +1205,13 @@ export class Tournament extends React.PureComponent<TournamentProperties, any> {
                                     : <span>
                                         <select className="rank-selection" value={tournament.settings.lower_bar} onChange={this.setLowerBar}>
                                             {ranks.map((r, idx) => (
-                                                <option key={idx} value={r.rank}>{r.label}</option>
+                                                <option key={idx} value={rank_long_string(r)}>{rank_long_string(r)}</option>
```

Comment:
Tournament creation currently does not work, `{"error":{"min_ranking":["A valid integer is required."],"max_ranking":["A valid integer is required."]}}` - related to these lines perhaps?